components/templates/src/global_fns/files.rs

```diff
@@ -104,7 +104,7 @@ impl TeraFn for GetUrl {
             // anything else
             let mut segments = vec![];
 
-            if lang != self.config.default_language {
+            if lang != self.config.default_language && !path.starts_with(&format!("/{}", lang)) {
```

Comment:
you can do that without allocating by slicing I believe?

Reply:
Can you elaborate please?

Reply:
Something like `path[1..].starts_with(lang)`. Haven't tried it so it might not work

Reply:
Indeed, this works! Thanks for the handy trick :)

I've amended the last commit.